src/utils/constants.ts

```diff
@@ -124,7 +124,11 @@ export const BALANCER_POOL_DATA_QUERIES_ADDRESSES: Record<number, Address> = {
     [ChainId.ZKEVM]: '0xF24917fB88261a37Cc57F686eBC831a5c0B9fD39',
 };
 
-export const BALANCER_VAULT = '0xBA12222222228d8Ba445958a75a0704d566BF2C8';
+export const BALANCER_VAULT: Address =
+    '0xBA12222222228d8Ba445958a75a0704d566BF2C8';
+
+export const BALANCER_RELAYER =
```

Comment:
This will likely have to be per network.

Reply:
You're right - I'll go ahead and update it to be per network 👍